draft-thomson-http-oblivious.md

```diff
@@ -381,6 +381,226 @@ Plaintext Messages support arbitrary length padding. Clients and servers MAY pad
 as needed to hide metadata leakage through ciphertext length.
 
 
+# HTTP Usage
+
+A client interacts with the oblivious proxy resource by constructing an
+encapsulated request.  This encapsulated request is included as the content of a
+POST request to the oblivious proxy resource.  This request MUST only contain
+those fields necessary to carry the encapsulated request: a method of POST, a
+target URI of the URI of the oblivious proxy resource, a header field containing
```

Comment:
```suggestion
target URI of the oblivious proxy resource, a header field containing
```

Reply:
Shouldn't the client also indicate the oblivious request resource URI (in a header or elsewhere)? If not, how does the proxy know where to forward the message?

Reply:
Oh, I see in #19 that it's assumed the proxy forwards to a fixed request resource, rather than letting the client choose (source routing style). Perhaps that can be the default if the request resource is omitted, allowing clients to control the path if desired? (That has security and policy implications though...)